src/shims.rs

```diff
@@ -43,16 +77,14 @@ impl TargetsMetadata {
 
     pub fn try_into(self) -> Result<tuf::TargetsMetadata> {
         if self.typ != tuf::Role::Targets {
-            bail!(ErrorKind::Encoding(format!(
-                "Attempted to decode targets metdata labeled as {:?}",
-                self.typ
-            )));
+            let msg = format!("Attempted to decode targets metadata labeled as {:?}", self.typ);
```

Comment:
This can be semi-ugly, but this is what `rustfmt` does, and IMO it makes it more consistent to have everyone just `cargo fmt` it before committing so we're all on the same page. That said, we can discuss later since this is completely a matter of taste and aesthetics.